snuba/datasets/transactions.py

```diff
@@ -137,6 +144,20 @@ def __init__(self):
             time_parse_columns=('start_ts', 'finish_ts')
         )
 
+    def _get_promoted_columns(self):
+        # TODO: Support promoted tags
```

Comment:
Should environment and release be in this list of promoted tags?

Reply:
possibly. I still need to think over the list of promoted tags. That's an orthogonal problem.